src/components/Navbar/Navbar.css

```diff
@@ -1,4 +1,49 @@
-nav {
+nav .pages-nav {
+  padding-top: 4rem;
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
+}
+
+nav .pages-nav a {
+  background-color: rgb(48, 48, 48);
+  padding: 14.5rem 6rem 4rem 6rem;
+  height: 40vh;
+  border-radius: 1rem;
+  color: white;
+  text-shadow: 1.5px 1.5px 1.5px black;
+  font-weight: 800;
+  font-size: xx-large;
+}
+
+nav .pages-nav a:hover {
+  animation: swing .9s step-end;
```

Comment:
This is cool. Random thought: maybe the hover effect could continue until the user mouses away from the link, instead of its current behavior?

https://github.com/user-attachments/assets/c349a50a-49b3-4e45-92c3-f3666adbdb41

Reply:
Ah ok, sounds good, I'll go ahead and work on an update! :D